src/index.html

```diff
@@ -15,8 +15,45 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="header_top">
+        <div class="header_logo">
+          <img src="images/logo.png" alt="MOYO-logo">
+        </div>
+        <div class="header_section">
+          <ul class="header_nav">
+            <li class="nav_item is-active">
+              <a href="">apple</a>
+            </li>
+            <li class="nav_item">
+              <a href="">samsung</a>
+            </li>
+            <li class="nav_item">
+              <a href="">Smartphones</a>
+            </li>
+            <li class="nav_item hover" data-qa="hover">
+              <a href="">Laptops & Computers</a>
+            </li>
+            <li class="nav_item">
+              <a href="">Gadgets</a>
+            </li>
+            <li class="nav_item">
+              <a href="">Tablets</a>
+            </li>
+            <li class="nav_item">
+              <a href="">Photo</a>
+            </li>
+            <li class="nav_item">
+              <a href="">Video</a>
+            </li>
+          </ul>
+        </div>
+      </div>
```

Comment:
Remember about correct indentation between parent and child elements. Each level of nesting should have a 2-space offset. Also, avoid blank lines between parent and child elements.